src/science/misc/cable_carbon.F90

```diff
@@ -150,8 +155,16 @@ SUBROUTINE carbon_pl(dels, soil, ssnow, veg, canopy, bgc)
     wbav = MAX( 0.01, wbav )  ! EAK Jan2011
 
     ! drought stress
-    coef_drght = EXP( 5.*( MIN( 1., MAX( 1., wbav**( 2 - soil%ibp2 ) - 1.) /    &
-         ( soil%swilt**( 2 - soil%ibp2 ) - 1. ) ) - 1. ) )
+    CampbellExp         = 2.0 - soil%ibp2
+    EffStressIndexWater = wbav**( CampbellExp ) - 1.0
+    EffStressIndexWater = MAX( 1.0, EffStressIndexWater )
+
+    EffStressIndexWilting = soil%swilt**( CampbellExp ) - 1.0
+    
+    RelativeStress =  EffStressIndexWater / EffStressIndexWilting
+    RelativeStress = MIN( 1.0, RelativeStress ) 
+    
+    coef_drght = EXP( 5.0 * ( RelativeStress  - 1.0 ) )
```

Comment:
I believe the "-1.0" is in the calculation of `RelativeStress` and not `coef_drght`:
```suggestion
    RelativeStress =  EffStressIndexWater / EffStressIndexWilting - 1.0
    RelativeStress = MIN( 1.0, RelativeStress ) 
    
    coef_drght = EXP( 5.0 * RelativeStress )
```
Only important because it should already be accounted for in the MIN(1, RelativeStress) call. But please, double check the parentheses count...

Reply:
DOH! Again - I have lost what I wrote in response. Strictly shouldn't matter  so long as RelativeStress is less than 1, but in principle if for e.g. 1 < RelativeStress < 2, then - 1 before the <MIN would change answer from non-sensical to sensical. Whether this is a good thing or not, OR how coef_cd below it is calibrated I have no idea.